src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -57,6 +57,9 @@ protected function executeCommand(array $command_args)
 
         $input = $this->input();
         $useTty = $input->isInteractive() ? null : false;
+        if ($input->isInteractive() && function_exists('posix_isatty') && !posix_isatty(STDOUT)) {
+            $useTty = false;
```

Comment:
You're assigning a bool inside an if statement here using something that's already been queried before. This can be simplified.

```bash
isInteractive | fnExists | !posix_isatty(stdout) | result
      T       |     T    |         T             |    F
      T       |     F    |         T             |    null
      T       |     T    |         F             |    null
      T       |     F    |         F             |    null
      F       |     T    |         T             |    F
      F       |     F    |         T             |    F
      F       |     T    |         F             |    F
      F       |     F    |         F             |    F

!isInteractive || (fnExists && !posix_isatty(stdout)) = F
isInteractive && (!fnExists || posix_isatty(stdout)) = null
```
so
`$useTty = $input->isInteractive && (!function_exists('posix_isatty') || posix_isatty(STDOUT)) ? null : false;`
Yeah?